tests/test_callbacks.py

```diff
@@ -146,3 +147,30 @@ def test_async_async_callbacks():
     t = Thread(target=test_async_callbacks)
     t.start()
     t.join()
+
+
+def test_callback_num_times(capsys):
```

Comment:
nit Unclear what testing overhead is for this for nominal usage vs. benchmark measurements.

Can you document expected order-of-magnitude runtime for this configured as-is?
e.g. "For average processor, expect about 0.2s for num_millions=1, repeats=2. Expect about 10s for num_millions=20, repeats=10."
(mebbe not super accurate, but good for gauging?)

Reply:
Thanks for the suggestion! I did it like this:
```
    # Example runtimes (Intel Xeon 2.2GHz, fully optimized):
    #   num_millions  1, repeats  2:  0.1 secs
    #   num_millions 20, repeats 10: 11.5 secs
```

Reply:
Looks great! Thank you!